tests/system/action/motion_change_recommendation/test_create.py

```diff
@@ -210,3 +214,12 @@ def test_create_change_recommendation_other_motion_not_colliding(self) -> None:
         self.create_change_recommendation(1, 5, motion_id=42)
         response = self.cr_request(1, 5)
         self.assert_status_code(response, 200)
+
+    def test_create_change_recommendation_collide_check(self) -> None:
+        self.create_change_recommendation(2, 5)
+        response = self.cr_request(1, 2)
+        self.assert_status_code(response, 400)
+        self.assertIn(
+            "The recommendation collides with an existing one",
+            response.json["message"],
+        )
```

Comment:
This test can be removed again as it is the same as min_overlap_2

Reply:
I removed this surplus test.